src/app.config.js

```diff
@@ -2,7 +2,7 @@ let config = {};
 let hostname = process.env.INCORE_REMOTE_HOSTNAME || "";
 
 config["hostname"] = hostname;
-config["semanticService"] = "";
+config["semanticServiceType"] = `${hostname}/semantics/api/types`;
```

Comment:
missing an ending "/" 
Or alternatively you can add that / when you do `${config.semanticServiceType}${data[key]}`